cmt/src/main/resources/META-INF/persistence.xml

```diff
@@ -21,15 +21,10 @@
         http://xmlns.jcp.org/xml/ns/persistence
         http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
    <persistence-unit name="primary">
-      <!-- If you are running in a production environment, add a managed 
-         data source, this example data source is just for development and testing! -->
-      <!-- The datasource is deployed as WEB-INF/cmt-quickstart-ds.xml, you
-         can find it in the source at src/main/webapp/WEB-INF/cmt-quickstart-ds.xml -->
-      <jta-data-source>java:jboss/datasources/CMTQuickstartDS</jta-data-source>
```

Comment:
Because it was removed from the WildFly quickstart. I basically applied most of the changes that were in this commit:
https://github.com/wildfly/quickstart/commit/235ce39ffa0e3fc24e43390a3a0b66df2bc33df2

This file is deprecated and the quickstart seems to work without it.